validators/structure_validator.py

```diff
@@ -5,8 +5,20 @@
 from validators.css_validator import CssValidator
 from utils.html_navigation import compare_content
 
+from typing import Tuple
 
-def compare(solution: str, submission: str, trans: Translator, **kwargs):
+
+def get_similarity(sol: str, sub: str) -> Tuple[float, float]:
+    from html_similarity import style_similarity, structural_similarity, similarity
```

Comment:
Isn't this an unused import?
```suggestion
    from html_similarity import style_similarity, structural_similarity
```